Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -506,25 +520,33 @@ def fetch_attributes_command(client: Client, params: Dict[str, str]):
     Returns: List of indicators.
 
     """
+    global LIMIT
     tlp_color = params.get('tlp_color')
     reputation = params.get('feedReputation')
     tags = argToList(params.get('attribute_tags', ''))
     feed_tags = argToList(params.get("feedTags", []))
     attribute_types = argToList(params.get('attribute_types', ''))
     query = params.get('query', None)
-    params_dict = clean_user_query(query) if query else build_params_dict(tags=tags, attribute_type=attribute_types, limit=2000,
-                                                                          page=1)
+    last_run = demisto.getLastRun().get('timestamp') if demisto.getLastRun() else ''
+    params_dict = clean_user_query(query, LIMIT, from_timestamp=last_run) if query else\
+        build_params_dict(tags=tags, attribute_type=attribute_types, limit=LIMIT, page=1, from_timestamp=last_run)
     search_query_per_page = client.search_query(params_dict)
+    demisto.debug(f'params_dict: {params_dict}')
     while len(search_query_per_page.get("response", {}).get("Attribute", [])):
         demisto.debug(f'search_query_per_page number of attributes:\
-                      {len(search_query_per_page.get("response", {}).get("Attribute", []))}\
-                        page: {params_dict["page"]}')
+                      {len(search_query_per_page.get("response", {}).get("Attribute", []))} page: {params_dict["page"]}')
         indicators = build_indicators(search_query_per_page, attribute_types, tlp_color, params.get('url'), reputation, feed_tags)
-        demisto.createIndicators(indicators)
+        if LIMIT > 2000:
+            for iter_ in batch(indicators, batch_size=2000):
+                demisto.createIndicators(iter_)
+        else:
+            demisto.createIndicators(indicators)
```

Comment:
```suggestion
        for iter_ in batch(indicators, batch_size=2000):
            demisto.createIndicators(iter_)
```
if LIMIT <= 2000 then the first batch will be the only one.
if you do it because LIMIT might change in the `clean_user_query()` method, see my note about this method below.